packages/cli/test/helpers/create-consumable-html.test.ts

```diff
@@ -1,85 +1,90 @@
+        'start': 1807839,
+        'duration': 973172
+      },
+      {
+        'phase': 'transition',
+        'start': 2781011,
+        'duration': 1540986
+      },
+      {
+        'phase': 'render',
+        'start': 4321997,
+        'duration': 1905528
+      },
+      {
+        'phase': 'paint',
+        'start': 6227525,
+        'duration': 33171
+      }
+    ]
+  }, {
+    'duration': 6260696,
+    'js': 5310439,
+    'phases': [
+      {
+        'phase': 'load',
+        'start': 0,
+        'duration': 1807839
+      },
+      {
+        'phase': 'boot',
+        'start': 1807839,
+        'duration': 973172
+      },
+      {
+        'phase': 'transition',
+        'start': 2781011,
+        'duration': 1540986
+      },
+      {
+        'phase': 'render',
+        'start': 4321997,
+        'duration': 1905528
+      },
+      {
+        'phase': 'paint',
+        'start': 6227525,
+        'duration': 33171
+      }
+    ]
+  }
+];
 
 describe('create-consumable-html test', () => {
   it(`resolveTitles()`, () => {
     const resolved = resolveTitles({
       // @ts-ignore
       servers: [{ name: 'Hello World' }, { name: 'Hello World 2' }],
-      plotTitle: 'Override',
+      plotTitle: 'Override'
     });
     expect(resolved.servers[0].name).to.equal('Control: Hello World');
     expect(resolved.servers[1].name).to.equal('Experiment: Hello World 2');
     expect(resolved.plotTitle).to.equal('Override');
   });
 
   it(`bucketPhaseValues()`, () => {
-    const testSamples = [
-      {
-        'duration': 6260696,
-        'js': 5310439,
-        'phases': [
-          {
-            'phase': 'load',
-            'start': 0,
-            'duration': 1807839,
-          },
-          {
-            'phase': 'boot',
-            'start': 1807839,
-            'duration': 973172,
-          },
-          {
-            'phase': 'transition',
-            'start': 2781011,
-            'duration': 1540986,
-          },
-          {
-            'phase': 'render',
-            'start': 4321997,
-            'duration': 1905528,
-          },
-          {
-            'phase': 'paint',
-            'start': 6227525,
-            'duration': 33171,
-          },
-        ]
-      }, {
-        'duration': 6260696,
-        'js': 5310439,
-        'phases': [
-          {
-            'phase': 'load',
-            'start': 0,
-            'duration': 1807839,
-          },
-          {
-            'phase': 'boot',
-            'start': 1807839,
-            'duration': 973172,
-          },
-          {
-            'phase': 'transition',
-            'start': 2781011,
-            'duration': 1540986,
-          },
-          {
-            'phase': 'render',
-            'start': 4321997,
-            'duration': 1905528,
-          },
-          {
-            'phase': 'paint',
-            'start': 6227525,
-            'duration': 33171,
-          },
-        ]
-      }
-    ];
     // @ts-ignore
```

Comment:
why do we have ts-ignore here? we should type the `const TEST_SAMPLES:  Sample[] = [`

Reply:
fixed!